src/RoslynTestKit/CompletionProviderFixture.cs

```diff
@@ -55,7 +55,7 @@ private static Action<ImmutableArray<CompletionItem>> CreateAssertionBasedOnExpe
 
         private void VerifyExpectations(Document document, IDiagnosticLocator locator, CompletionTrigger? trigger, Action<ImmutableArray<CompletionItem>> assertion)
         {
-            var selectedTrigger = trigger ?? CompletionTrigger.Default;
+            var selectedTrigger = trigger ?? CompletionTrigger.Invoke;

```

Comment:
Can you explain the purpose/meaning of that change?

Reply:
`CompletionTrigger.Default` is obsolete, they recommend to use `CompletionTrigger.Invoke` instead.